### PR TITLE
Update hi.po

### PR DESCRIPTION
**Pull request by gostsdmitry**

### CODE REVIEW

**Comment by gostsdmitry** [comment]
I'm sorry, please. Tell me for what reason the error appears. She wasn't there 2 days ago...
error: process completed with exit code 2

**Comment by mwleeds** [comment]
`hi.po:690: number of format specifications in 'msgid' and 'msgstr' does not match`

File: po/hi.po [hunk 1]
```diff
@@ -690,7 +690,7 @@ msgstr "अमान्य --require-version तर्क: %s"
 #: app/flatpak-builtins-build-finish.c:533
 #, c-format
 msgid "Too few elements in --extra-data argument %s"
-msgstr "अतिरिक्त डेटा तर्क में बहुत कम तत्व %s"
+msgstr "--extra-data तर्क %s . में बहुत कम तत्व %s"
```
Comment (mwleeds) [inline po/hi.po:693]: I think the issue is you have `%s`  twice here whereas the untranslated string has it once.

Comment (gostsdmitry) [reply po/hi.po]: Thanks! This is where the problem lies.
This is currently the last patch for the Hindi translation.

**Pull request merged**